airbyte-integrations/connectors/source-snowflake/acceptance-test-config.yml

```diff
@@ -5,3 +5,19 @@ tests:
   spec:
     - spec_path: "src/test-integration/resources/expected_spec.json"
       config_path: "secrets/config.json"
+  connection:
```

Comment:
Please use the latest acceptance-test-config format. [Example here.](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-zendesk-support/acceptance-test-config.yml#L1). I'd be curious to check the test output under `high` test strictness level, we might get this for free 😄